components/Modal/Action.vue

```diff
@@ -414,9 +412,29 @@ export default {
         default:
       }
     },
+    setNetworkFees(type) {
```

Comment:
Seems the function returns value only
```suggestion
    calculateNetworkFees(type) {
```